setup.py

```diff
@@ -1240,20 +1254,25 @@ def build_cmake(build_ext, ext, prefix, plugin_ext=None, options=None):
     extdir = os.path.abspath(
         os.path.dirname(build_ext.get_ext_fullpath(ext.name)))
     config = 'Debug' if build_ext.debug else 'Release'
-    cmake_args = [
-        '-DUSE_MPI=ON',
-        '-DCMAKE_BUILD_TYPE=' + config,
-        '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}'.format(config.upper(), extdir),
-        '-DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_{}={}'.format(config.upper(),
-                                                        lib_output_dir),
-    ]
+
+    # Pass additional compiler flags by setting CMAKE_CXX_FLAGS_[DEBUG/RELEASE]
+    # so that cmake will append these flags to CMAKE_CXX_FLAGS
+    additional_cxx_flags = ' '.join(additional_flags)
```

Comment:
Do you need to pipes.quote that, to ensure it's in " "s?

Reply:
Yes, that's much more secure.

Reply:
But I use a string prefix in the bash args, I assume they are of the same functionality?